packages/@okta/vuepress-theme-default/assets/css/okta.scss

```diff
@@ -2,7 +2,7 @@
 
 @import
 
-  '~@okta/nim/src/scss/nim',
```

Comment:
Why is it that we are getting rid of importing all of nim and instead individual items?

Reply:
Alas, because we need to insert a variable definition in the middle of nim.

If we define our variable before, nim's definition overrides it. If we define it after, we're too late and it will never apply.

This could be avoided if the design system declared variables with `!default`: https://github.com/okta/odyssey/issues/37

(The design system doesn't yet have fast velocity and regular releases, so I think we should work around it on our side for now.)